doc/astspec.txt

```diff
@@ -118,8 +126,53 @@ Concrete syntax:
 AST:
 
 .. code-block:: nim
-  nnkInfix(nnkIdent(!"&"), nnkStrLit("abc"), nnkStrLit("xyz"))
+  nnkInfix(
+    nnkIdent(!"&"), 
+    nnkStrLit("abc"), 
+    nnkStrLit("xyz")
+  )
+
+Note that with multiple infix operators, the command is parsed by operator 
+precedence.
```

Comment:
Seems to be true
```nim
Infix
    Ident !"and"
    Infix
      Ident !"+"
      Infix
        Ident !"+"
        IntLit 5
        Infix
          Ident !"*"
          IntLit 3
          IntLit 4
      IntLit 6
    IntLit 2
```
puts `and` at the top of the tree, which matches up with [syntax precedence](http://nim-lang.org/docs/manual.html#syntax-precedence)